lib/ansible/modules/remote_management/redfish/idrac_redfish_facts.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2019 Dell EMC Inc.
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: idrac_redfish_facts
+version_added: "2.8"
+short_description: Manages servers through iDRAC using Dell Redfish APIs
+description:
+  - Builds Redfish URIs locally and sends them to remote iDRAC controllers to
+    get information back.
+  - For use with Dell iDRAC operations that require Redfish OEM extensions
+options:
```

Comment:
In a follow up PR could you please ensure all `remote_management/redfish` modules have `type:` set for the options?
A single PR which updates all modules would help improve the online documentation

Reply:
Will do. Once all modules are merged I can submit one PR to update all modules in `remote_management/redfish`. Thanks.

Reply:
Thanks, @gundalow and @jose-delarosa. Good idea. FYI - I created issue #54688 for this.